science/2021-06-15-science-blog-1/index.md

```diff
@@ -7,7 +7,7 @@ author: CWA Team
 layout: science
 ---
 
-An English version of this science blog article will follow in due course.
+An English version of this science blog article will follow in due course. Please take a look at the [German version](https://www.coronawarn.app/de/science/2021-06-15-science-blog-1/) until the English version is available.

```

Comment:
It would be better to use a relative URL here since the target is on the same webserver.

Reply:
I suggest changing
"[German version](https://www.coronawarn.app/de/science/2021-06-15-science-blog-1/)" to
"[German version](../../de/science/2021-06-15-science-blog-1)"

Thanks in any case for the cross-linking! 👍🏻

Reply:
Done in https://github.com/corona-warn-app/cwa-website/pull/1397/commits/1529bafea7eea61c63cb8d78183a7cefbd7c108c.

Thanks for the reminder!